src/bin/index.js

```diff
@@ -329,25 +357,54 @@ yargs(hideBin(process.argv))
             ...commonOptions,
           },
           async (argv) => await lightnetExplorer(argv)
-        )
-        .demandCommand();
-    }
-  )
-  .version(
-    fs.readJsonSync(path.join(__dirname, '..', '..', 'package.json')).version
-  )
-  .alias('h', 'help')
-  .alias('v', 'version')
-
-  .epilog(
-    // chalk.reset is a hack to force the terminal to retain a line break below
-    chalk.reset(
-      `
-
-    █▄ ▄█ █ █▄ █ ▄▀▄
-    █ ▀ █ █ █ ▀█ █▀█
+        );
+    },
+  };
+}
 
-     Mina Protocol
-      `
-    )
-  ).argv;
+function getCustomizedStrings() {
+  // https://github.com/yargs/yargs/issues/199
```

Comment:
Could you give more detail on why you linked these in the comments? Is this indicating an on-going issue, an example of how to provide usage strings?

Reply:
@MartinMinkov these comments have been in the code base for a long time. https://github.com/o1-labs/zkapp-cli/blob/d4203f1b173e1ac93aeb8c5dfdb71d8ff8a1fdf7/src/bin/index.js#L45-L46 I think @shimkiv moved some logic into the `getCustomizedStrings` function and left the comments that were already there. 

I'm not sure why the comments are there, and I don't see the value in leaving them there unless someone has an explanation to add more clarity.